src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -84,6 +85,27 @@
         return emails;
     }
     
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code student}
+     * and {@code instructor}.
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmails(
+            FeedbackSessionAttributes session, StudentAttributes student,
+            InstructorAttributes instructor) {
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        EmailWrapper email = new EmailWrapper();
+        
+        if (instructor != null) {
+            email = generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(course, session,
+                    instructor);
+        }
+        if (student != null) {
+            email = generateFeedbackSessionEmailBaseForStudentSubmissionConfirmation(course, session,
+                    student);
+        }
+        return email;
```

Comment:
Both student and instructor cannot be non-null, as only one person will be submitting the response, whether it be instructor or student ?
If both student and instructor are null. What to do in this case ? we don't want to send email if not instructor OR student ?